e2e/test/iothub/messaging/FaultInjectionPoolAmqpTests.MessageReceiveFaultInjectionPoolAmqpTests.cs

```diff
@@ -633,39 +41,10 @@ public async Task Message_DeviceSak_TcpConnectionLossReceiveUsingCallbackRecover
                 .ConfigureAwait(false);
         }
 
-        [LoggedTestMethod, Timeout(LongRunningTestTimeoutMilliseconds)]
-        public async Task Message_IoTHubSak_TcpConnectionLossReceiveUsingCallbackRecovery_MultipleConnections_Amqp()
-        {
-            await ReceiveMessageUsingCallbackRecoveryPoolOverAmqpAsync(
-                    TestDeviceType.Sasl,
-                    Client.TransportType.Amqp_Tcp_Only,
-                    PoolingOverAmqp.MultipleConnections_PoolSize,
-                    PoolingOverAmqp.MultipleConnections_DevicesCount,
-                    FaultInjection.FaultType_Tcp,
-                    FaultInjection.FaultCloseReason_Boom,
-                    authScope: ConnectionStringAuthScope.IoTHub)
```

Comment:
For callback tests, removed the tests that use hub admin credentials to connect as the device. How they authenticate should have no bearing on how the C2D links or device connects are detected to be killed and how the SDK re-initializes.